Tangem/App/ViewModels/CardViewModel.swift

```diff
@@ -425,13 +468,38 @@ class CardViewModel: Identifiable, ObservableObject {
 
     func didScan() {
         Analytics.logScan(card: cardInfo.card, config: config)
-        tangemSdkProvider.setup(with: config.sdkConfig)
+        onAppear()
```

Comment:
```suggestion
        updateSdkConfig()
```
пока в `onAppear()` нет никакой дополнительной логики нужной для `didScan()`, думаю тут лучше поставить `updateSdkConfig()`, а то получается, что если ты отсканировал карточку, значит обязательно что-то появилось, но что?..

Reply:
Обновил